modules/navigation/godot_navigation_server.cpp

```diff
@@ -808,8 +814,117 @@ void GodotNavigationServer::process(real_t p_delta_time) {
 			active_maps_update_id[i] = new_map_update_id;
 		}
 	}
+
+#ifdef MODULE_NAVIGATION_ENABLED
+	_process_path_queries();
+#endif // MODULE_NAVIGATION_ENABLED
+}
```

Comment:
I would add a comment or preferably rename `GodotNavigationServer::process()` to make it clear it is being done on a physics tick, not a frame. The name `process` in godot is associated with frame processing (rightly or wrongly), so this would be good to not have confused. Perhaps `physics_process()`, to be consistent with other methods.

Reply:
I agree with the process() but that raises the general problem that the NavigationServer uses a function called `process()` for its main iteration that is only called with the physics_process tick.

Reply:
Presumably it's only called from the engine internally, so should be fine to change. But it's no biggy if the main API function doesn't have this name, but I would suggest either using `physics_process` as much as possible, instead of `process`, or use another term like `update`, which is less confusing.

Reply:
Maybe a general term like update is better cause if it is no longer the physics tick responsible for this function it would be confusing and require renames.

Reply:
Yup `update` is fine, and possibly better.